test/server_integration/http/platform/headers.ts

```diff
@@ -47,17 +46,15 @@ export default function ({ getService }: FtrProviderContext) {
       const agent = new Http.Agent({
         keepAlive: true,
       });
-      const { protocol, hostname, port } = Url.parse(
-        getUrl.baseUrl(config.get('servers.opensearchDashboards'))
-      );
+      const { protocol, hostname, port } = config.get('servers.opensearchDashboards.fullURL');
 
       function performRequest() {
         return new Promise((resolve, reject) => {
           const req = Http.request(
             {
               protocol,
               hostname,
-              port,
+              port: parseInt(port, 10),
```

Comment:
why do we need to specify port `10` now?

Reply:
The port in URL is a string, and we need parse it to a number. There isn't a specific port 10. 10 is a radix here, to ensure this parse result will use 10 hexadecimal.